third-party/realdds/src/dds-device.cpp

```diff
@@ -189,19 +191,19 @@ void dds_device::open( const dds_stream_profiles & profiles )
     _impl->open( profiles );
 }
 
-void dds_device::set_option_value( const std::shared_ptr< dds_option > & option, float new_value )
+void dds_device::set_option_value( const std::shared_ptr< dds_option > & option, json new_value )
 {
     wait_until_ready( 0 );  // throw if not
-    _impl->set_option_value( option, new_value );
+    _impl->set_option_value( option, std::move( new_value ) );
```

Comment:
Can `new_value` be moved? impl function takes by value

Reply:
Sure: `new_value` isn't `const`. We're done with it here, so we can move it to the next function to handle. Moving it is a performance optimization, so that the next function doesn't have to copy it.